src/components/DataTable/DataTable.component.tsx

```diff
@@ -73,7 +75,9 @@ export const DataTable = <Data extends BaseRowData = BaseRowData>({
                 rowData={rowData}
               />
             ))}
-          {!isInitialLoad && data.length === 0 && <TableEmpty />}
+          {!isInitialLoad && data.length === 0 && tableEmptyMessage && (
```

Comment:
maybe let's add it some default value for this props so that this check won't be necessary?